BREAKING.md

```diff
@@ -254,7 +397,78 @@ The `inputmode` property for `ion-searchbar` now defaults to `undefined`. To get
 
 #### Segment
 
-<!-- TODO https://gist.github.com/brandyscarney/e6cfe43c359bb2c932e12f8d615e1669 -->
+Segment was completely revamped to use the new iOS design including an all new gesture that applies for both Material Design & iOS. Due to these changes, some breaking changes were inevitably introduced in order to support the new design.
+
+##### Button States
+
+- The activated styles and properties have been removed. These are no longer being used in the latest spec as the indicator and ripple are used to show activation. Properties removed:
```

Comment:
```suggestion
- The activated styles and variables have been removed. These are no longer being used in the latest spec as the indicator and ripple are used to show activation. Variables removed:
```

Reply:
I'll update this to "custom CSS properties" since that's what we call them in our docs